docs/src/implemented-proposals/rent.md

```diff
@@ -24,11 +24,11 @@ Even if those processes are out of scope of rent collection, all of manipulated
 
 ## Actual processing of collecting rent
 
-Rent is due for one epoch's worth of time, and accounts always have `Account::rent_epoch` of `current_epoch + 1`.
+Rent is due for one epoch's worth of time, and accounts have `Account::rent_epoch` of `current_epoch` or `current_epoch + 1` depending on the rent regime.
```

Comment:
@rwalker-apple Hi, I'm yet again tweaking the rent protocol a bit, mainly affected by this dos possibility: #10847. Could you review this in your free time?